tox.ini

```diff
@@ -1,41 +1,40 @@
 [tox]
 envlist =
-    {py3.5,py3.6,py3.7,py3.8}-{sqlite,postgresql}-trytond-{5.0,5.2,5.4,5.6,latest}
+    {py3.5,py3.6,py3.7,py3.8,py3.9}-{sqlite,postgresql}
     linters
 
 [testenv]
 deps =
     .
-    {py3.5,py3.6,py3.7,py3.8}-postgresql: psycopg2-binary
-    trytond-latest: trytond
-    trytond-5.6: trytond>=5.6,<5.7
-    trytond-5.4: trytond>=5.4,<5.5
-    trytond-5.2: trytond>=5.2,<5.3
-    trytond-5.0: trytond>=5.0,<5.1
+    {py3.5,py3.6,py3.7,py3.8,py3.9}-postgresql: psycopg2-binary
+    trytond>=5.4,<5.5
```

Comment:
Is this necessary right now? If tox is installing the package to run the tests (it should by default if I'm not missing anything) a proper trytond version should be autoinstalled as a dependency of this very module version!

When releasing future module versions only the tryton.cfg file will need to be bumped!